File: src/main/java/com/trade/securities/application/port/in/GetFscStockSubscriptionRightUseCase.java
package com.trade.securities.application.port.in;

import java.util.Map;

public interface GetFscStockSubscriptionRightUseCase {
    Map<String, Object> getSubscriptionRights(int pageNo, int numOfRows, String basDt, String itmsNm,
            String likeItmsNm);
}

File: src/main/kotlin/com/trade/securities/application/port/in/GetFscStockSubscriptionRightUseCase.kt
package com.trade.securities.application.port.`in`

interface GetFscStockSubscriptionRightUseCase {
    fun getSubscriptionRights(pageNo: Int, numOfRows: Int, basDt: String, itmsNm: String?, likeItmsNm: String?): Map<String, Any?>
}
